### DISCUSSION TITLE
Cannot find module '@ioc:Adonis/Core/Env' or its corresponding type declarations.

### DISCUSSION THREAD

**Discussion by osamfrimpong** [adonisjs/core#4435] [Help]
I'm using V6.

import Env from '@ioc:Adonis/Core/Env' this gives the error Cannot find module '@ioc:Adonis/Core/Env' or its corresponding type declarations.

**Comment by leofmarciano** [comment]
In version 6, some imports were changed.

The core's `env` is located at `'@adonisjs/core/env'`, however, I do not recommend using it directly. It is best to utilize the typing available in `start/env`.

Read more about it at https://docs.adonisjs.com/guides/environment-variables
